.github/workflows/php.yml

```
@@ -0,0 +1,24 @@
+name: PHP tests
+on: [pull_request]
```

Like Travis, we run tests when the PR are merged

```suggestion
on: [push, pull_request]
```